xbmc/addons/PVRClient.h

```diff
@@ -723,9 +723,41 @@ namespace PVR
     bool IsPlayingEncryptedChannel(void) const;
     bool IsPlayingRecording(void) const;
     bool IsPlaying(void) const;
-    CPVRRecordingPtr GetPlayingRecording(void) const;
+
+    /*!
+     * @brief Set the channel that is currently playing.
+     * @param channel The channel that is currently playing.
+     */
+    void SetPlayingChannel(const CPVRChannelPtr channel);
```

Comment:
Without the other pieces for new pvr streaming being in place I'm unable to do this "right" nowadays.